solidity/contracts/core/pool-diamond/facets/PerpTradeFacet.sol

```diff
@@ -495,9 +495,15 @@ contract PerpTradeFacet is PerpTradeFacetInterface {
     } else {
       if (ds.shortSizeOf[indexToken] == 0)
         ds.shortAveragePriceOf[indexToken] = vars.price;
-      else
+      else {
         ds.shortAveragePriceOf[indexToken] = GetterFacetInterface(address(this))
-          .getNextShortAveragePrice(indexToken, vars.price, sizeDelta);
+          .getNextShortAveragePriceInt(
+            indexToken,
+            vars.price,
+            int256(sizeDelta),
+            0
```

Comment:
could you please confirm if it can be passed as `0`, since when you need to calculate the nextDelta
you need to use realizedPnL along with delta from shortSize

Reply:
Confirmed. `increasePosition` does not realize any PnL. So, this must always be 0.

Reply:
noted